protocol-specs/generators/multisignature_registration/index.js

```diff
@@ -14,164 +14,206 @@
 
 'use strict';
 
-const {
-	hexToBuffer,
-	intToBuffer,
-	hash,
-	bufferToHex,
-	signData,
-} = require('@liskhq/lisk-cryptography');
+const { signData } = require('@liskhq/lisk-cryptography');
+const { Codec } = require('@liskhq/lisk-codec');
 const BaseGenerator = require('../base_generator');
 
-const networkIdentifier =
-	'e48feb88db5b5cf5ad71d93cdcd1d879b6d5ed187a36b0002cc34e0ef9883255';
+const codec = new Codec();
+
+const networkIdentifier = Buffer.from(
+	'e48feb88db5b5cf5ad71d93cdcd1d879b6d5ed187a36b0002cc34e0ef9883255',
+	'hex',
+);
```

Comment:
This `networkIdentifier` is shared across almost all of the generators. Can we move such shared stuff to a common file to reduce the code footprint so we can review and manage it easily.

Reply:
yes but with the current structure, all of the generator is independent, and even if we change the network identifier and it still works